cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -76,6 +77,7 @@ can be removed automatically by this program.`)
 
 	// Settings for limits
 	maxConnections = flag.Uint64("max_connections", 0, `If provided, the maximum number of connections to establish before refusing new connections. Defaults to 0 (no limit)`)
+	fdRlimit       = flag.Uint64("fd_rlimit", expectedFDs, `Sets the rlimit on the number of open file descriptors for the proxy to the provided value. If set to zero, disables attempts to set the rlimit. Defaults to a value which can support 4K connections to one instance`)
```

Comment:
Hmm... sorry for making you do more work but I had a better idea: now that `--fd_rlimit` disables attempting to call syscall.Setrlimit, it'd be cool if we set the default value for windows to 0 and otherwise set it to 8500; then, put the error back in the "windows" code. That way, by default, windows users won't see an error, but if they try to set the variable they will see a message indicating that there is no support.

The easiest way to do that is probably to just put the `expectedFDs` constant in the `limits` package., and have the windows version just be defined to 0.

Reply:
Done.